src/services/bunker_cases/midterm_slashing_penalty.py

```diff
@@ -183,29 +183,30 @@ def predict_midterm_penalty_in_frame(
                 ref_epoch, all_slashed_validators, EpochNumber(midterm_penalty_epoch)
             )
             penalty_in_frame += MidtermSlashingPenalty.get_validator_midterm_penalty(
-                validator, len(bound_slashed_validators), total_balance
+                validator, bound_slashed_validators, total_balance
             )
         return Gwei(penalty_in_frame)
 
     @staticmethod
     def get_validator_midterm_penalty(
         validator: LidoValidator,
-        bound_slashed_validators_count: int,
-        total_balance: Gwei
+        bound_slashed_validators: list[Validator],
+        total_balance: Gwei,
     ) -> Gwei:
         """
         Calculate midterm penalty for particular validator
-        https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#slashings
+        https://github.com/ethereum/consensus-specs/blob/dev/specs/electra/beacon-chain.md#modified-process_slashings
         """
-        # We don't know which balance was at slashing epoch, so we make a pessimistic assumption that it was 32 ETH
-        slashings = Gwei(bound_slashed_validators_count * MAX_EFFECTIVE_BALANCE)
+        # We don't know validators effective balances on the moment of slashing,
+        # so we assume that it was at least `effective_balance`
+        slashings = Gwei(sum(int(v.validator.effective_balance) for v in bound_slashed_validators))
```

Comment:
This could break compatibility with current version.
Depending on hardfork version it should use one alghorithm or another